src/structures/CommandInteraction.js

```diff
@@ -32,6 +32,12 @@ class CommandInteraction extends Interaction {
      * @type {boolean}
      */
     this.deferred = false;
+    
+    /**
+     * Whether the reply to this interaction is ephemeral
+     * @type {boolean}
+     */
+    this.ephemeral = false;
```

Comment:
Maybe initialize it with `null` since the interaction wasn't replied to yet?

Reply:
Okay